chapters/00-Einfuehrung.Rmd

```diff
@@ -33,42 +33,42 @@ Jetzt können Sie R schon als einfachen Taschenrechner benutzen!
 Addition:
 
 ```{r}
-3+2
+3 + 2
 ```
 
 Subtraktion:
 
 ```{r}
-3-2
+3 - 2
 ```
 
 Multiplikation:
 
 ```{r}
-3*2
+3 * 2
 ```
 
 Division:
 
 ```{r}
-3/2
+3 / 2
 ```
 
 Beachten Sie dabei, dass Sie auch Klammern setzen können:
 
 
 ```{r}
-(3+2)*5
+(3 + 2) * 5
```

Comment:
Thanks for adding the spaces. It's something we talked about in the beginnings of the book but we still have multiple inconsistencies across chapters. There are other "operations" of this kind in the book. Would you mind going through the chapters to fixing this?